src/App.js

```diff
@@ -1,39 +1,69 @@
-import { useState } from 'react'
+import { useState, useEffect } from 'react'
 import styled from 'styled-components/macro'
 import { cities } from './capstone.json'
 import AttractionCard from './components/AttractionCard/AttractionCard'
+import CreateForm from './components/CreateForm/CreateForm'
 import Filter from './components/Filter/Filter'
+import TripCard from './components/TripCard/TripCard'
 
 export default function App() {
   const [userInput, setUserInput] = useState('')
   const [likedPlaces, setLikedPlaces] = useState([])
+  const [cards, setCards] = useState(loadFromLocal('cards') ?? [])
+
+  useEffect(() => {
+    saveToLocal('cards', cards)
+  }, [cards])
 
   return (
-    <>
-      <AppLayout>
-        <Filter userInput={userInput} setUserInput={setUserInput} />
-        {cities.map(({ name, attraction, id }) => (
-          <CardLayout key={id}>
-            <h2>{name}</h2>
-            {attraction
-              .filter(item =>
-                item.name.toLowerCase().includes(userInput.toLowerCase().trim())
-              )
-              .map(({ name, image }) => (
-                <AttractionCard
-                  key={name}
-                  name={name}
-                  image={image}
-                  onLike={handleLikePlace}
-                  likedPlaces={likedPlaces}
-                />
-              ))}
-          </CardLayout>
-        ))}
-      </AppLayout>
-    </>
+    <AppLayout>
+      <CreateForm onCreateTrip={CreateTrip} />
```

Comment:
Da es sich um ein Formular handelt, würde ich hier als Namen onSubmit vorschlagen

Reply:
Wenn du mal in meine CreateForm Komponente schaust, ist dort meine onSubmit Funktion enthalten. Hier in der App reiche ich wiederum meine CreateTrip Funktion weiter! :)